### ISSUE TITLE
Article

### ISSUE THREAD

**Issue by AmirhosseinnnKhademi** [okada39/pinn_cavity#4]
Thanks for sharing such nice work. Have you reported your job through any article?? Because I cannot find any article related to this job!

**Comment by okada39** [comment]
> Thanks for sharing such nice work. Have you reported your job through any article?? Because I cannot find any article related to this job!

We did not report any articles about this work, because its algorithms had been published in some articles as shown in README. We just implemented them using TensorFlow.

**Comment by AmirhosseinnnKhademi** [comment]
Thanks for the clarification. So, if I am not mistaken, you have reimplemented the PINN code, done by M. Raissi, for the 2D cavity flow?

**Comment by okada39** [comment]
> Thanks for the clarification. So, if I am not mistaken, you have reimplemented the PINN code, done by M. Raissi, for the 2D cavity flow?

Yes. We reimplemented some parts. More strictly, we developed them as a scratch code according to algorithms.

**Comment by AmirhosseinnnKhademi** [comment]
Thanks. It is helpful and useful
